src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -192,4 +192,25 @@ public static Status parseStatus(String status) throws ParseException {
         }
         return new Status(trimmedStatus);
     }
+
+    /**
+     * Parses a {@code String} delivery attribute to ensure it is a valid delivery attribute.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code String} delivery attribute is invalid.
+     */
+    public static String parseAttribute(String attribute) throws ParseException {
+        requireNonNull(attribute);
+        String trimmedAttribute = attribute.trim();
+        if (trimmedAttribute.equals("address")
+            || trimmedAttribute.equals("cost")
+            || trimmedAttribute.equals("date")
+            || trimmedAttribute.equals("eta")
+            || trimmedAttribute.equals("id")
+            || trimmedAttribute.equals("status")) {
+            return trimmedAttribute;
+        } else {
+            throw new ParseException("Current attributes supported are: address, cost, date, eta, id, status");
```

Comment:
Might want to consider moving the exception message to a final static variable instead. 
For example to such format 
```
public static final String MESSAGE_ATTRIBUTES_UNSUPPORTED = "Current attributes supported are: address, cost, date, eta, id, status";
```

Reply:
This has been fixed!